client/locations-list-map.js

```diff
@@ -206,7 +206,11 @@ function createFilters(data) {
 
   const filters = {
     states: {},
+    entryAge:{},
   };
+  filters.entryAge.placeholder = "Entry age";
```

Comment:
I think the public phrase could "Last Updated" here :)

Reply:
Gotcha, will change this when we go back and make our edits!